arangodb-net-standard/ViewApi/ViewsApiClient.cs

```diff
@@ -0,0 +1,215 @@
+                    var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    return DeserializeJsonFromStream<GetViewPropertiesResponse>(stream);
+                }
+                throw await GetApiErrorException(response).ConfigureAwait(false);
+            }
+        }
+
+        /// <summary>
+        /// Partially changes properties of a view
+        /// PATCH /_api/view/{view-name}/properties
+        /// </summary>
+        /// <param name="viewNameOrId">The name or identifier of the view.</param>
+        /// <param name="body">The body of the request containing required properties.</param>
+        /// <returns></returns>
+        public virtual async Task<ViewResponse> PatchViewPropertiesAsync(string viewNameOrId, ViewDetails body)
+        {
+            string uri = $"{_apiPath}/{viewNameOrId}/properties";
+            var content = GetContent(body, new ApiClientSerializationOptions(true, true));
+            using (var response = await _transport.PatchAsync(uri, content).ConfigureAwait(false))
+            {
+                if (response.IsSuccessStatusCode)
+                {
+                    var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    return DeserializeJsonFromStream<ViewResponse>(stream);
+                }
+                throw await GetApiErrorException(response).ConfigureAwait(false);
+            }
+        }
+
+        /// <summary>
+        /// Changes all properties of a view
+        /// PUT /_api/view/{view-name}/properties
+        /// </summary>
+        /// <param name="viewName">The name of the view.</param>
+        /// <param name="body">The body of the request containing required properties.</param>
+        /// <returns></returns>
+        public virtual async Task<ViewResponse> PutViewPropertiesAsync(string viewName, ViewDetails body)
+        {
+            string uri = $"{_apiPath}/{viewName}/properties";
+            var content = GetContent(body, new ApiClientSerializationOptions(true, true));
+            using (var response = await _transport.PutAsync(uri, content).ConfigureAwait(false))
+            {
+                if (response.IsSuccessStatusCode)
+                {
+                    var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    return DeserializeJsonFromStream<ViewResponse>(stream);
+                }
+                throw await GetApiErrorException(response).ConfigureAwait(false);
+            }
+        }
+
+        /// <summary>
+        /// Renames a view
+        /// PUT /_api/view/{view-name}/rename
+        /// </summary>
+        /// <param name="viewName">The name of the view.</param>
+        /// <param name="body">The body of the request containing required properties.</param>
+        /// <returns></returns>
+        public virtual async Task<PutRenameViewResponse> PutRenameViewAsync(string viewName, PutRenameViewBody body)
```

Comment:
The doc mentions that this method isn't available in a cluster. We could mention that in the XML doc comments.

https://www.arangodb.com/docs/stable/http/views-arangosearch.html#rename-a-view
> Note: This method is not available in a cluster.

Reply:
I've added it.